### DISCUSSION TITLE
How to show code revision

### DISCUSSION THREAD

**Discussion by longjulia** [Jo-Lo/Tafe22S203#6] [General]
Hey everyone! How can I see the detailed code revisions from show log? Like when you see a side by side view of the code from one commit to another? When I show log, I can see my comment and the files modified, but how about the actual lines of code?

**Comment by Jo-Lo** [comment]
When in show log highlight any commit message and in the bottom window where it lists all filles which have been added or modified.
Double click on any certain file and it would display a new window showing differences.

hopefully my instructions are clear, let me know if all goes well.
